references/bibtex/hh.bib

```diff
@@ -120879,7 +120879,7 @@ @phdthesis{s:Bradley:Jicaltepec-Mixtec:1965
     superseded = {s:Bradley:Jicaltepec-Mixtec}
 }
 
-@misc{dial:BradleyBradley:Sso,
+@misc{dial:BradleyBradley:Melokwo,
```

Comment:
@d97hah There's another `dial:BradleyBradley:Sso` below, so I renamed this one with the language name used here.